opossum-tool/OpossumUI
# Mark previously preferred signals

### Summary of changes

We mark previously preferred signals by a grey star. If a previously preferred attribution is marked as preferred again, we only show the golden star to indicate that it is preferred now, not the grey one.

### Context and reason for change

A signal which had been marked as preferred in a previous run should not be considered as preferred again automatically. But still we want to highlight these signals.

### How can the changes be tested

Open the file opossum_input.json. In audit view, choose the index.tsx file, check that one of the attributions is marked with a grey star. Then mark this attribution as preferred and click confirm; then the grey  star should be replaced by a golden one.
Choose the file ElectronBackend/Types/types.ts and check that the suggested signal "Vue, 4.1.1" has the grey star. Add this signal and check that the attribution also has the grey star.
